app/services/memberships/revoke_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Memberships
+  class RevokeService < BaseService
+    def revoke(id)
```

Comment:
```suggestion
    def call(id)
```

Then we can have `Memberships::RevokeService.call()` instead of `Memberships::RevokeService.revoke()`

Reply:
Is that a convention? I find it more explicit to name the method but happy to change and follow any good practice

Reply:
It's not a convention, but I like having a `call` method in all services if the naming is like this.

`Memberships::RevokeService.revoke` is quite redundant, I prefer having `MembershipsService.revoke` or `Memberships::RevokeService.call`

Reply:
Updated here and all invocations